src/controllers/routes.ts

```diff
@@ -5,17 +5,29 @@ import { pingHandler } from './handlers/ping-handler'
 import { statusHandler } from './handlers/status-handler'
 import { commsSceneHandler } from './handlers/comms-scene-handler'
 import { muteHandler } from './handlers/mute-handler'
+import { addSceneAdminHandler, removeSceneAdminHandler, listSceneAdminsHandler } from './handlers/scene-admin-handlers'
+import { wellKnownComponents as authVerificationMiddleware } from '@dcl/platform-crypto-middleware'
 
 // We return the entire router because it will be easier to test than a whole server
-export async function setupRouter(_globalContext: GlobalContext): Promise<Router<GlobalContext>> {
+export async function setupRouter({ components }: GlobalContext): Promise<Router<GlobalContext>> {
   const router = new Router<GlobalContext>()
   router.use(errorHandler)
```

Comment:
This is the error middleware I was referring before, I think it is safe to remove all catches from handlers and let this guy handle the throwings